internal/command/postgres/import.go

```diff
@@ -202,19 +202,8 @@ func resolveImportCommand(ctx context.Context) string {
 		dataOnly = flag.GetBool(ctx, "data-only")
 	)
 
-	importCmd := "migrate "
-	if noOwner {
-		importCmd = importCmd + " -no-owner"
-	}
-	if clean {
-		importCmd = importCmd + " -clean"
-	}
-	if create {
-		importCmd = importCmd + " -create"
-	}
-	if dataOnly {
-		importCmd = importCmd + " -data-only"
-	}
-
-	return importCmd
+	return fmt.Sprintf(
+		"migrate -no-owner=%v -clean=%v -create=%v -data-only=%v",
+		noOwner, create, clean, dataOnly,
```

Comment:
`clean` and `create` flag order is swapped

Reply:
Yikes, thanks for catching this!